lib/internal/process/promises.js

```diff
@@ -7,13 +7,37 @@ const pendingUnhandledRejections = [];
 const asyncHandledRejections = [];
 let lastPromiseId = 0;
 
+const kPromiseRejectWithNoHandler = 0;
```

Comment:
Make a mention that this needs to be kept in sync with code in `src/bootstrapper.cc`.